android/expoview/build.gradle

```diff
@@ -65,7 +65,7 @@ repositories {
 }
 
 android {
-  compileSdkVersion safeExtGet("compileSdkVersion", 28)
+  compileSdkVersion safeExtGet("compilesdkversion", 29)
```

Comment:
was it intentional to change `compileSdkVersion` to `compilesdkversion` in a bunch of places in this commit?

Reply:
Oops, good catch, it wasn't intentional, sorry 😅